lib/phoenix_live_view/engine.ex

```diff
@@ -618,11 +625,22 @@ defmodule Phoenix.LiveView.Engine do
 
   defp changed_assigns(assigns) do
     checks =
-      for {key, _} <- assigns, not nested_and_parent_is_checked?(key, assigns) do
+      for {{changed_var, key}, _} <- assigns, not nested_and_parent_is_checked?(key, assigns) do
+        changed =
+          case changed_var do
+            :changed ->
+              quote do
+                changed
+              end
```

Comment:
I think this is the same as:

```suggestion
              Macro.var(:changed, __MODULE__)
```

Which means you can do `changed = Macro.var(changed_var, __MODULE__)`?